lib/openstudio-standards/btap/structure.rb

```diff
@@ -564,18 +542,93 @@ def initialize(model = nil, cat = "commerce", lload = 30)
         mass.setSpace(space)
       end
 
-      # @todo
-      @co2 = 0
+      # Embodied CO2-e kg (A1-A3) of a model is tallied separately as follows:
+      @co2              = {}
+      @co2[:structure ] = 0
+      @co2[:insulation] = 0
+      @co2[:cladding  ] = 0
+
+      # The :structure key/value pair includes above grade 'structures' (e.g.
+      # slabs, columns) and 'framing' (e.g. wood-framed vs steel-framed). Why
+      # grouped together? In many smaller-scale facilities, structure and
+      # framing are synonymous, or at least tightly coupled, e.g.:
+      #   - load-bearing wood-framed walls in small-scale residential
```

Comment:
First round of embodied carbon tallies is limited to structure and framing, as a single number - explains why.